packages/compiler/src/renderers/components/ImageView/index.tsx

```diff
@@ -0,0 +1,119 @@
+/******************************************************************
+ * created by lqy at 06/16/2021
+ ******************************************************************/
+
+import React from 'react';
+import Modal from '@material-ui/core/Modal';
+import Fade from '@material-ui/core/Fade';
+import { makeStyles, createStyles } from '@material-ui/styles';
+
+export interface PreviewProps {
+    children?: React.ReactElement;
```

Comment:
It's better to follow the guide from https://github.com/typescript-cheatsheets/react#useful-react-prop-type-examples.

```suggestion
    children?: React.ReactNode;
```